src/components/SettlementButton.tsx

```diff
@@ -133,6 +134,13 @@ function SettlementButton({
         PaymentMethods.openWalletPage();
     }, []);
 
+    const policy = ReportUtils.getPolicy(policyID);
+    const session = useSession();
+    const chatReport = ReportUtils.getReport(chatReportID);
+    const isPaidGroupPolicy = ReportUtils.isPaidGroupPolicy(iouReport as OnyxEntry<Report>) || ReportUtils.isPaidGroupPolicyExpenseChat(chatReport as OnyxEntry<Report>);
```

Comment:
We can check only `isPaidGroupPolicyExpenseChat` since `isPaidGroupPolicy` is already called in `isPaidGroupPolicyExpenseChat`:

```suggestion
    const isPaidGroupPolicy = ReportUtils.isPaidGroupPolicyExpenseChat(chatReport as OnyxEntry<Report>);
```